src/pages/main/toolbar/service/resource-service.ts

```diff
@@ -0,0 +1,52 @@
+import { Resource } from '@model';
+
+let testResourceList: Resource[] = Array(30).fill(0).map((_, index) => {
+  return {
+    projectId: 'project_xyz',
+    resourceId: `resource_${index+1}`,
+    name: `image_${index+1}.png`
+  };
+});
+
+/* Upload resource and return ID of new created resource */
+export const createResource = async (resourceName: string): Promise<string> => {
+  // TODO: integrate
+  await new Promise(res => {
+    setTimeout(res, 1000);
+  });
+
+  const newResource: Resource = {
+    projectId: 'project_xyz',
+    resourceId: `resource_${resourceName}`,
+    name: `Name: ${resourceName}`
+  };
+
+  // testResourceList = [...testResourceList, newResource];
```

Comment:
what is this

Reply:
Whole this file will be rewritten, so let's keep it that way